cli/cmd/vuln_host.go

```diff
@@ -949,13 +875,23 @@ func hostScanPackagesVulnDetailsTable(vulns []api.HostScanPackageVulnDetails) []
 			fixedVersion = vuln.FixInfo.FixedVersion
 		}
 
+		vuln.FixInfo.FixedVersion = fixedVersion
```

Comment:
ugh, I might need to understand why are we changing the API response here 🤔 , I don't think we should do that, could we sync up?

Reply:
Good catch Salim, I think you're right. In fact I think 872-877 is redundant and can just be removed.